CHANGELOG.md

```diff
@@ -1462,7 +1462,7 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   plugin post-install hooks for performance reasons.  
   [Samuel Giddins](https://github.com/segiddins)
 
-* Change shell script relative paths to use `${PODS_ROOT}` instead of 
+* Change shell script relative paths to use `${PODS_ROOT}` instead of
```

Comment:
this one actually needs two empty spaces in the end for markdown formatting to work.